tools/autograd/templates/python_torch_functions.cpp

```diff
@@ -413,6 +413,9 @@ static PyObject * TypeError_to_NotImplemented_(PyObject* self, PyObject* args, P
 
 ${py_methods}
 
+// XXX: ops that are bound here are not exposed to the C++ api nor the JI2T.
+// Any new ops added here should be accompanied with a commment why they are not
+// being registered through native_functions.yaml, and include reviwers from C++/JIT
```

Comment:
nit: reviwers -> reviewers

Maybe just write that people should attempt to add the JIT and cpp labels? Contributors might not know who to ping